### ISSUE TITLE
[BUG] Read file slow with alluxioruntime

### ISSUE THREAD

**Issue by robin1900** [fluid-cloudnative/fluid#1127]
**What is your environment(Kubernetes version, Fluid version, etc.)**

Fluid 0.60  alluxioruntime   , use lustre as backend storage

**Describe the bug**

run in pod with dataset bound

time cp alluxio-2.6.2-bin.tar.gz /dev/null 

real    1m36.357s
user    0m0.014s
sys     0m1.047s

the  file is about 1.8GB ,  why read file is so slow for the first time

---
apiVersion: data.fluid.io/v1alpha1
kind: AlluxioRuntime
metadata:
  name: lustre
spec:
  # Add fields here
  replicas: 3
  data:
    replicas: 2

  tieredstore:
    levels:
      - mediumtype: MEM
        path: /dev/shm
        quota: 30Gi
        high: "0.9"
        low: "0.6"
      - mediumtype: SSD
        path: /data/alluxio
        quota: 150Gi
        high: "0.9"
        low: "0.7"

  properties:
    # jni-fuse related configurations
    alluxio.fuse.jnifuse.enabled: "true"
    alluxio.user.client.cache.enabled: "false"
    alluxio.user.client.cache.store.type: MEMORY
    alluxio.user.client.cache.dir: /alluxio/ram
    alluxio.user.client.cache.page.size: 2MB
    alluxio.user.client.cache.size: 1800MB
    # alluxio master
    alluxio.master.metastore: ROCKS
    alluxio.master.metastore.inode.cache.max.size: "10000000"
    alluxio.master.journal.log.size.bytes.max: 500MB
    alluxio.master.metadata.sync.concurrency.level: "128"
    alluxio.master.metadata.sync.executor.pool.size: "128"
    alluxio.master.metadata.sync.ufs.prefetch.pool.size: "128"
    # alluxio configurations
    alluxio.user.block.worker.client.pool.min: "512"
    alluxio.fuse.debug.enabled: "false"
    alluxio.web.ui.enabled: "false"
    alluxio.user.file.writetype.default: THROUGH
    alluxio.user.ufs.block.read.location.policy: alluxio.client.block.policy.LocalFirstPolicy
    alluxio.user.block.write.location.policy.class: alluxio.client.block.policy.LocalFirstAvoidEvictionPolicy
    alluxio.worker.allocator.class: alluxio.worker.block.allocator.GreedyAllocator
    alluxio.user.block.size.bytes.default: 16MB
    alluxio.user.streaming.reader.chunk.size.bytes: 32MB
    alluxio.user.local.reader.chunk.size.bytes: 32MB
    alluxio.worker.network.reader.buffer.size: 32MB
    alluxio.worker.file.buffer.size: 320MB
    alluxio.user.metrics.collection.enabled: "false"
    alluxio.master.rpc.executor.max.pool.size: "1024"
    alluxio.master.rpc.executor.core.pool.size: "128"
    #alluxio.master.mount.table.root.readonly: "true"
    alluxio.user.update.file.accesstime.disabled: "true"
    alluxio.user.file.passive.cache.enabled: "false"
    alluxio.user.block.avoid.eviction.policy.reserved.size.bytes: 2GB
    alluxio.master.journal.folder: /journal
    alluxio.master.journal.type: UFS
    alluxio.user.block.master.client.pool.gc.threshold: 2day
    alluxio.user.file.master.client.threads: "1024"
    alluxio.user.block.master.client.threads: "1024"
    alluxio.user.file.readtype.default: CACHE
    alluxio.security.stale.channel.purge.interval: 365d
    alluxio.user.metadata.cache.enabled: "true"
    alluxio.user.metadata.cache.expiration.time: 2day
    alluxio.user.metadata.cache.max.size: "1000000"
    alluxio.user.direct.memory.io.enabled: "true"
    alluxio.fuse.cached.paths.max: "1000000"
    alluxio.job.worker.threadpool.size: "164"
    alluxio.user.worker.list.refresh.interval: 2min
    alluxio.user.logging.threshold: 1000ms
    alluxio.fuse.logging.threshold: 1000ms
    alluxio.worker.block.master.client.pool.size: "1024"
  master:
    jvmOptions:
      - "-Xmx6G"
      - "-XX:+UnlockExperimentalVMOptions"
      - "-XX:ActiveProcessorCount=8"
  worker:
    jvmOptions:
      - "-Xmx12G"
      - "-XX:+UnlockExperimentalVMOptions"
      - "-XX:MaxDirectMemorySize=32g"
      - "-XX:ActiveProcessorCount=8"
    resources:
      limits:
        cpu: 8
  fuse:
    env:
      MAX_IDLE_THREADS: "32"
    jvmOptions:
      - "-Xmx16G"
      - "-Xms16G"
      - "-XX:+UseG1GC"
      - "-XX:MaxDirectMemorySize=32g"
      - "-XX:+UnlockExperimentalVMOptions"
      - "-XX:ActiveProcessorCount=24"
    resources:
      limits:
        cpu: 16
    args:
      - fuse
      - --fuse-opts=kernel_cache,rw,max_read=131072,attr_timeout=7200,entry_timeout=7200,nonempty

**Comment by cheyang** [comment]
Do you have tried to copy this file from lustre to local file system? How many time does it take?

**Comment by robin1900** [comment]
> Do you have tried to copy this file from lustre to local file system? How many time does it take?

yes . only take 2 seconds

**Comment by cheyang** [comment]
You can check throughput with `https://docs.alluxio.io/os/user/stable/en/operation/Admin-CLI.html#metrics`.  Are they using the same machine to copy file, and did you clear page cache. Actually , If it took 2 seconds to copy 1.8 GiB file, it means nearly 1 GiB/second.  Can you confirm this?

**Comment by robin1900** [comment]
> You can check throughput with `https://docs.alluxio.io/os/user/stable/en/operation/Admin-CLI.html#metrics`. Are they using the same machine to copy file, and did you clear page cache. Actually , If it took 2 seconds to copy 1.8 GiB file, it means nearly 1 GiB/second. Can you confirm this?


yes, I cleaned zhe page cache. it takes 4 seconds from lustre.  the lustre is small cluster with 3 nodes , and base on ssd. so it is fast

**Comment by robin1900** [comment]
> You can check throughput with `https://docs.alluxio.io/os/user/stable/en/operation/Admin-CLI.html#metrics`. Are they using the same machine to copy file, and did you clear page cache. Actually , If it took 2 seconds to copy 1.8 GiB file, it means nearly 1 GiB/second. Can you confirm this?

I got the metrics:

Master.DirectoriesCreated  (Type: COUNTER, Value: 566)
Master.EdgeCacheEvictions  (Type: COUNTER, Value: 0)
Master.EdgeCacheHits  (Type: COUNTER, Value: 26,913,392)
Master.EdgeCacheLoadTimes  (Type: COUNTER, Value: 29,690,148)
Master.EdgeCacheMisses  (Type: COUNTER, Value: 493,752)
Master.EdgeCacheSize  (Type: GAUGE, Value: 123,438)
Master.EdgeLockPoolSize  (Type: GAUGE, Value: 123,439)
Master.FileBlockInfosGot  (Type: COUNTER, Value: 0)
Master.FileInfosGot  (Type: COUNTER, Value: 123,459)
Master.FilesCompleted  (Type: COUNTER, Value: 122,872)
Master.FilesCreated  (Type: COUNTER, Value: 122,872)
Master.FilesFreed  (Type: COUNTER, Value: 0)
Master.FilesPersisted  (Type: COUNTER, Value: 0)
Master.FilesPinned  (Type: GAUGE, Value: 0)
Master.FreeFileOps  (Type: COUNTER, Value: 0)
Master.GetAcl.UFS:%2Fjournal%2FMetricsMaster.UFS_TYPE:local  (Type: TIMER, Value: 565)
Master.GetAcl.User:root.UFS:%2Fjournal%2FMetricsMaster.UFS_TYPE:local  (Type: TIMER, Value: 1)
Master.GetBlockMasterInfo.User:root  (Type: TIMER, Value: 245)
Master.GetFileBlockInfoOps  (Type: COUNTER, Value: 0)
Master.GetFileInfoOps  (Type: COUNTER, Value: 11)
Master.GetFileLocations.User:root.UFS:%2Fjournal%2FMetricsMaster.UFS_TYPE:local  (Type: TIMER, Value: 123,791)


it seams that too many GetFileLocations.  I just copies one file and only one time

**Comment by cheyang** [comment]
I suggest you to repeat the experiment and check `Cluster.BytesReadUfsThroughput (Type: GAUGE, Value: 0B/MIN)`. Thanks. @apc999  Any sugggestions.
